Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -255,6 +255,44 @@ public async Task<IActionResult> UpdateCurrentPaymentMethod(string accountname,
             }
         }
 
+        [Authorize(Policies.PROVISORY_USER_OR_SUPER_USER)]
+        [HttpGet("/accounts/{accountname}/declined-invoices")]
```

Comment:
What do you think about 

```suggestion
        [HttpGet("/accounts/{accountname}/invoices/declined")]
```

Or 

```suggestion
        [HttpGet("/accounts/{accountname}/invoices?declined=true")]
```